apps/services/user-notification/infra/user-notification.ts

```diff
@@ -39,12 +39,12 @@ const getEnv = (services: {
   AUTH_DELEGATION_MACHINE_CLIENT_SCOPE: json([
     '@island.is/auth/delegations/index:system',
   ]),
-  USER_NOTIFICATION_APP_PROTOCOL: {
```

Comment:
wont this break the `env-switcher` support in the test app?

Reply:
This was not being used anywhere in the notification service. The app registered a device token, so I assume that this is some legacy code as the test app is just registered with device token in dev environment.

Reply:
this is part of the callback url schema in Cognito that we use for the env switcher in the test app.

Callback configuration in Cognito

```
is.island.app.dev://cognito
is.island.app.staging://cognito
is.island.app://cognito
```

Reply:
will be fixed with deeplinks, resolving.